UnityProject/Assets/Scripts/Core/Chat/ChatRelay.cs

```diff
@@ -304,29 +304,16 @@ public void AddMentorPrivMessageToClient(string message)
 
 		switch (channels)
 		{
-			case ChatChannel.Local:
-				break;
-			case ChatChannel.Action:
-				break;
-			case ChatChannel.Examine:
-				break;
-			case ChatChannel.Ghost: //(Max) : we need a sound for ghosts
-				break;
-			case ChatChannel.System:
-				break;
-			case ChatChannel.Warning:
-				break;
-			case ChatChannel.Blob:
-				break;
-			case ChatChannel.OOC:
-				break;
 			case ChatChannel.Syndicate:
 				_ = SoundManager.Play(Chat.Instance.commonSyndicteChannelSound);
 				break;
 			case ChatChannel.Security:
```

Comment:
These two cases haven't been updated to use `HasFlag()`. Speaking of which, from what I recall using `HasFlag()` has performance issues - might be allocating or something. Probs not too much of an issue given we're using it throughout chat but something to keep in mind.

Reply:
HasFlag boxes the values twice. That was an issue a while back with the IsInLayer method used in a hot loop that caused dozens to hundreds of megabytes of garbage. Probably not going to be an issue with chat.

Reply:
We can always go safe and use bitwise operations instead

Reply:
i didn't want to create any more problems by using bitwise operators since i know ill screw it up somehow so i chose the safest path